src/cli-main.ts

```diff
@@ -365,7 +360,7 @@ program
       await updateHostnamesAndRules(requestHostnames, rules, sandboxId, sandboxPropertyId);
       console.log(`Successfully updated sandbox_id: ${sandboxId} sandbox_property_id: ${sandboxPropertyId}`);
     } catch (e) {
-      console.log(e);
+      cliUtils.logAndExit(1, e);
```

Comment:
To be consistent, cases where logging just the exception should be prepended with `ERROR: `